components/AppNav.tsx

```diff
@@ -115,7 +115,7 @@ const AppNav: React.FC<{}> = () => {
   const { data } = useGetAppQuery()
   useEffect(() => {
     if (data && data.session) {
-      setSession(data.session)
+      setSession(data.session as Session)
```

Comment:
Why is this needed? Apollo hook returns properly typed `GetAppQuery | undefined`, after check for `data&&data.session` typescript should infer proper type by itself.

Reply:
Ah, ignore this PR. The more recent submissions and challenge one takes priority. 

I probably needed to add annotation because typescript was yelling at me and that was the only way to make it disappear